### DISCUSSION TITLE
Custom Titlebar - Cannot Drag the Window

### DISCUSSION THREAD

**Discussion by egebilecen** [tauri-apps/tauri#5886] [Q&A] [answered]
Hello all. 

I am trying to create an application with custom title bar. I have followed [this](https://tauri.app/v1/guides/features/window-customization) guide but I am unable to make the application draggable. I have added `data-tauri-drag-region` to html element as well but still no luck.  I am building the application using `ReactJS`.
Any help is appreciated, thanks!

![Screenshot_1](https://user-images.githubusercontent.com/29331682/208924270-e023972b-f309-4a42-9d59-376d99c61a2c.png)

**Comment by FabianLars** [comment, marked as answer]
2 things that should make it work:
1) enable window.startDragging in the allowlist
2) the data attribute must be applied to the actual click target - often the innermost element (you often can find it by looking at which element is reported by `document.addEventListener('click', console.log)`) -> this is because otherwise the buttons inside the element would not work anymore for example.

**Reply by egebilecen** [reply]
I already had everything allowed. 2nd suggestion did the trick, thank you! It might be worth to considering adding that suggestion as a warning to guide.

**Reply by mijorus** [reply]
Worked for me too, thank you
